src/main/kotlin/org/rust/ide/intentions/DestructureIntention.kt

```diff
@@ -47,6 +49,9 @@ class DestructureIntention : RsElementBaseIntentionAction<DestructureIntention.C
     }
 
     private fun handleStruct(project: Project, editor: Editor, patIdent: RsPatIdent, struct: RsStructItem) {
+        val (toImport, _) = patIdent.getTypeReferencesInfoFromTy(struct.declaredType)
+        patIdent.importElements(toImport)
```

Comment:
Hm, looks like we always do the same actions: call `getTypeReferencesInfoFrom*` and call `importElements`
Maybe we can create a single public function to make both actions inside it?

Reply:
Also, I don't like that current functions are extention functions of `RsElement` because we use element only as context.
So I'd prefer pass element as explicit argument.
If you don't want to create new top-level function(s), let's create some object class like `RsImportHelper` and move them into it

Reply:
And maybe it's worth to call `import` method after main action?
Imagine the following case: our new awesome import feature throws an exception. And in this case, main action won't be executed and user doesn't see any change. Otherwise, we make all necessary actions and just miss imports that user can import manually in case of potential exception